run-tests.php

```diff
@@ -2629,11 +2621,7 @@ function run_test($php, $file, $env)
             $wanted_re = str_replace('%c', '.', $wanted_re);
             // %f allows two points "-.0.0" but that is the best *simple* expression
         }
-/* DEBUG YOUR REGEX HERE
-        var_dump($wanted_re);
-        print(str_repeat('=', 80) . "\n");
-        var_dump($output);
-         */
+
```

Comment:
I don't think this should be dropped.

Reply:
Hmm. I'd obviously prefer remove it. It's a slippery slope allowing hardcore debug code in the codebase.

Reply:
It's okay to drop, I think.